airflow/dag_processing/manager.py

```diff
@@ -263,7 +269,9 @@ def deactivate_stale_dags(
             # last_parsed_time is the processor_timeout. Longer than that indicates that the DAG is
             # no longer present in the file. We have a stale_dag_threshold configured to prevent a
             # significant delay in deactivation of stale dags when a large timeout is configured
-            dag_file_path = DagFileInfo(path=dag.fileloc, bundle_name=dag.bundle_name)
+            dag_file_path = DagFileInfo(
+                path=dag.fileloc, bundle_name=dag.bundle_name, bundle_version=dag.bundle_version
```

Comment:
I don't think we want to have the version here - we want to see if the dag is still present in the "latest" (a.k.a none) version.